config/locales/client.sv.yml

```diff
@@ -4179,8 +4179,6 @@ sv:
         problems_found: "Några förslag baserat på dina nuvarande webbplatsinställningar"
         new_features:
           title: "\U0001F381 Nya funktioner"
-          dismiss: "Avfärda"
```

Comment:
For next time you don't need to delete all these manually, they will get deleted when the translations next get updated :)